addons/sale/static/src/js/variant_mixin.js

```diff
@@ -382,6 +382,31 @@ var VariantMixin = {
                 }
             });
         }
+        // combination exclusions: array of array of ptav
+        // for example a product with 3 variation and one specific variation is disabled (archived)
+        //  requires the first 2 to be selected for the third to be disabled
+        if (combinationData.combination_exclusions) {
+            combinationData.combination_exclusions.forEach((excludedCombination) => {
+                const ptavDifference = excludedCombination.filter((ptav) => !combination.includes(ptav));
+                if (ptavDifference.length == 0) {
```

Comment:
I know this makes no difference in this context but ...

```suggestion
                if (ptavDifference.length === 0) {
```